app/test_main.py

```diff
@@ -1,3 +1,33 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zero_if_zero_input() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_should_return_zero_if_less_than_cat_year() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_minimum_non_zero_result() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_max_one_year() -> None:
```

Comment:
This test case could be parameterized along with similar cases to reduce redundancy .